account_asset_management/models/account_asset_profile.py

```diff
@@ -177,7 +179,10 @@ def _selection_method_time(self):
         Install the 'account_asset_management_method_number_end' to enable the
         'Number' and 'End' Time Methods.
```

Comment:
Please update the outdated comment.

Reply:
I will just remove it. In fact, that module never existed, that why we are adding it here.

Reply:
Know thy odoo history, young man :-)
https://github.com/OCA/account-financial-tools/tree/10.0/account_asset_management_method_number_end

Reply:
ok, thanks haha